### ISSUE TITLE
[BUG]: toolID undefined-vundefined

### ISSUE THREAD

**Issue by EvilKarter** [IBMa/equal-access#2352]
### Project

accessibility-checker for Node

### Operating system

Other

### Description

I have upgraded my npm dependeny `"accessibility-checker": "3.1.78"` to `"accessibility-checker": "4.0.8"`. 
But now I in my result I get `"toolID":"undefined-vundefined"` instead of  the expected `"toolID": "accessibility-checker-v4.0.8"`.

### Steps to reproduce

My achecker.yml: 
```
ruleArchive: versioned
policies:
    - IBM_Accessibility
failLevels:
    - violation
    - potentialviolation
reportLevels:
    - violation
    - potentialviolation
    - recommendation
    - potentialrecommendation
    - manual
    - pass
outputFormat: json
outputFolder: /path/to/output/
outputFilenameTimestamp: false
```
